Makefile

```diff
@@ -35,6 +35,9 @@ imgs_server_dest := $(imgs_source:%=build/server/%)
 
 raven_source := $(shell node -e 'console.log(require.resolve("raven-js/dist/raven.js"))')
 
+l10n_source := $(wildcard locales/*/*.properties)
+l10n_dest := $(l10n_source:%/*.properties=addon/webextension/_locales/%/messages.json)
```

Comment:
This doesn't look right to me, but I'm not sure.  What should the value of $(l10n_source) be?  If it's like `locales/en-US/something.properties locales/es-ES/something.properties` and you want to get `addon/webextensions/_locales/en-US/messages.json addon/webextensions/_locales/en-US/messages.json` then it's right.  This presumes that each directory has only one properties file and that it's name isn't important.